concourse/steps/scan_container_images.py

```diff
@@ -440,14 +441,9 @@ def print_protecode_info_table(
 
 
 def retrieve_buildlog(uuid: str):
-    concourse_cfg = concourse.util._current_concourse_config()
-    uam_cfg_name = concourse_cfg.concourse_uam_cfg()
-    concourse_uam_cfg = ctx.cfg_factory().concourse_uam(uam_cfg_name)
-
     pipeline_metadata = concourse.util.get_pipeline_metadata()
-    client = concourse.client.from_cfg(
-        concourse_cfg=concourse_cfg,
-        concourse_uam_cfg=concourse_uam_cfg,
+    client = ccc.concourse.client_from_cfg(
+        cfg_set=ctx.cfg_factory(),
```

Comment:
are you sure this will work (I did not check, so it might)? cfg_factory does not define default values (as opposed to cfg-sets)

Reply:
that's no problem the factory doesn't need any defaults